ui/client/actions/esp/process/editEdge.js

```diff
@@ -0,0 +1,16 @@
+import * as GraphUtils from "../../../components/graph/GraphUtils"
+import HttpService from "../../../http/HttpService"
+
+export function editEdge(process, before, after) {
```

Comment:
Maybe just edge, group, node, metrics, just like "comment"? Similar actions can be added in the future